tasks/kernel/build.sh

```diff
@@ -38,29 +50,29 @@ fi
 if ! common/host/fetch_cache_obj.sh "$INSTALL_OBJECT_KEY" "$INSTALL"; then
     export KBUILD_OUTPUT INSTALL
     (
-        set +e
         cd "$SOURCE"
 
         install -d "${INSTALL}/boot"
         INSTALL_MOD_PATH="${INSTALL}" make modules_install
 
-        if [[ "$DS_TARGET_ARCH" == "armel" || "$DS_TARGET_ARCH" == "armhf" ]]; then
```

Comment:
The script originally had this check; it is entirely gone now, since Kconfig enforces which architectures can use these configs.